Sources/VLCLibraryViewController.m

```diff
@@ -39,6 +40,20 @@
 
 @implementation EmptyLibraryView
 
+- (id)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    if (self) {
+        NSArray *subviewArray = [[NSBundle mainBundle] loadNibNamed:@"VLCEmptyLibraryView" owner:self options:nil];
```

Comment:
initiWithFrame is actually the wrong place to load from the nib. You should take a look how it's done everywhere else. You're also calling super and assigning it to self, then checking for nil and then assigning self = subviewArray.firstObject; without having the nil check again. I missed that the first time this needs to be done differently